docs/develop-guide.rst

```diff
@@ -4,44 +4,33 @@ Cider Developer Guide
 1 How to build
 --------------
 
-1.1 Build Modular SQL
+1.1 Build BDTK
 ~~~~~~~~~~~~~~~~~~~~~
 
 | **Prerequisite:**
 | Install docker and set proxy according to the guidance in:
 | https://docs.docker.com/config/daemon/systemd/
 | https://docs.docker.com/network/proxy/
 
-1. Clone repo locally
+1. Get the BDTK Source
 
 ::
 
-   # WORKDIR: ${workdir}
-   # velox-plugin
-   git clone https://github.com/intel-innersource/frameworks.ai.modular-sql.velox-plugin.git
-   pushd frameworks.ai.modular-sql.velox-plugin
-   git submodule update --init --recursive
-   popd
+   $ git clone --recursive https://github.com/intel/BDTK.git
+   $ cd BDTK
+   # if you are updating an existing checkout
+   $ git submodule sync --recursive
+   $ git submodule update --init --recursive
 
-1. Build an image from a Dockerfile
+2. Setting up BDTK develop envirenmont on Linux Docker
 
-::
+   We provide Dockerfile to help developers setup and install BDTK dependencies.
 
-   # WORKDIR: ${path_to_velox_plugin}/ci/docker
-   docker build -t ${image_name} .
+::
 
-   # Note: If you are behind an HTTP or HTTPS proxy server, for example in corporate settings,
-   #       you need to add this configuration in the Docker systemd service file, 
-   #       please follow the official guide: https://docs.docker.com/network/proxy/
-   # example:
-   # --build-arg HttpProxyHost=your.proxy.server --build-arg HttpsProxyHost=your.proxy.server --build-arg HttpProxyPort=your.proxy.server --build-arg HttpsProxyPort=your.proxy.port
-   
-   # Note: If the apt network still doesn't work, it may be a timezone problem. Here are two solutions:
-   # 1. sync system time with a independent service.
-   #    - config 'NTP' /etc/systemd/timesyncd.conf, you need to find a service in your company or elsewhere.
-   #    - run 'systemctl restart systemd-timesyncd.service'
-   # 2. add parameters after apt-get in Dockerfile to skip timezone check, like:
-   #    RUN apt-get -o Acquire::Check-Valid-Until=false -o Acquire::Check-Date=false update 
+   # Build an image from a Dockerfile
+   $ cd ${path_to_source_of_bdtk}/ci/docker
+   $ docker build -t ${image_name} .
 
    # Start a docker container for development
    docker run -d --name ${container_name} --privileged=true -v ${path_to_velox_plugin}:/workspace/velox-plugin ${image_name} /usr/sbin/init
```

Comment:
velox-plugin  --> BDTK

Reply:
do I need to change all the "velox-plugin" to BDTK?